worldedit_commands/init.lua

```diff
@@ -465,6 +465,30 @@ local check_sphere = function(name, param)
 		worldedit.player_notify(name, "invalid usage: " .. param)
 		return nil
 	end
+	if nil ~= area_protection.areas then

```

Comment:
No need for yoda style.  Also, you can just use `if area_protection.areas then`.